server/routes/aws/cloudwatch_tasks.py

```
@@ -0,0 +1,423 @@
+                                    session_id, task.id,
+                                )
+                        except Exception as exc:
+                            logger.exception("[CLOUDWATCH][ALARM] Failed to trigger RCA: %s", exc)
+
+        except Exception as db_exc:
+            logger.exception("[CLOUDWATCH][ALARM] DB error for user %s: %s", user_id, db_exc)
+
+    except Exception as exc:
+        logger.exception("[CLOUDWATCH][ALARM] Failed to process alarm payload")
+        if not user_id:
+            raise self.retry(exc=exc)
```

Ruff (0.15.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)